### DISCUSSION TITLE
Looking for a way to *easily* scale a sprite

### DISCUSSION THREAD

**Discussion by UnexpectedMaker** [Bodmer/TFT_eSPI#3082] [Q&A - Sprites]
Hey folks - I'm wanting to animate a arbitrary sprite getting bigger - think zooming in a static UI etc.

is there a "known way" to do this other than a complicated loop of reading the pixel data of the main full sized sprite and re-sizing it on the fly into smaller dynamically created sprites, where each gets bigger and bigger until it's the full sprite size?

I can't find anything on google on here about a better way to do it.

Cheers!
Seon

**Comment by UnexpectedMaker** [comment]
No one with any ideas on this?

**Reply by Maxmoon56** [reply]
Did you found something? Search how to program complex GUI that are not simple squares with works is hard

**Comment by TangerineDreamer** [comment]
hello,UnexpectedMaker ,it's about gesture,the first one i found:
https://www.codeproject.com/Articles/5287929/Swipe-Gestures-Using-a-TFT-Display
